src/themes/dark-theme/dark-theme-with-move.js

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
"dark-theme-with-move" contains an Uber-specific font name. I recommend reorganizing the themes to distinguish between the default themes and the themes specific to Uber. I also recommend renaming the theme folder/files to "uber-theme-light" and "uber-theme-dark" considering we want to eventually add the brand colors to the theme.

Reply:
I see your point, and it would be nicer to have a single theme with that includes custom font, icons, and maybe additional props, not sure if we want to have it as a part of public package tho. @gergelyke what do you think about it?